config/routes.rb

```diff
@@ -2,6 +2,8 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
+  resources :comments, only: [:new, :create, :edit, :update, :destroy]
+  get 'comments/destroy'
```

Comment:
This is not an idiomatic REST resource. GET paths should not modify/create/destroy resources